src/action-menu/ActionMenu.jsx

```diff
@@ -0,0 +1,146 @@
+import React, { useRef } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import classNamesBind from 'classnames/bind';
+import ThemeContext from 'terra-theme-context';
+import ContentContainer from 'terra-content-container';
+import Header from './ActionMenuHeader';
+import {
+  generateOnKeyDown,
+  itemByDirection,
+  itemByChar,
+  flattenActionItems,
+} from './_ActionUtils';
+
+import styles from './ActionMenu.module.scss';
+
+const cx = classNamesBind.bind(styles);
+
+const propTypes = {
+  /**
+   * Label for the ActionMenu.
```

Comment:
Should it mention this value will be used in the header if it's displayed?

Reply:
I'll rename this to `label`, which is what we've landed on for props that will be visible text.